base/docs/helpdb/Base.jl

```diff
@@ -207,7 +207,10 @@ isnan
 """
     println(x)
 
-Print (using [`print`](:func:`print`)) `x` followed by a newline.
+Print (using [`print`](:func:`print`)) `x` followed by a newline. To print to a specific output
```

Comment:
Thanks! Though, instead of mentioning the two-argument form in the description, it should be shown in the signature at the top, as `println(io::IO=STDOUT, x)`. This is much shorter and clearer.

By the way, it looks like `x` should be `x...` there too. Though the docs for `write` don't use ellipsis either. Anybody knows whether it could be intended?